aikau/src/main/resources/alfresco/services/UploadService.js

```diff
@@ -232,6 +232,7 @@ define(["dojo/_base/declare",
             // Store a response topic to publish on when the upload is complete...
             // This allows for scoped responses to be processed without pollution...
             this.currentResponseTopic = payload.alfResponseTopic;
+            this.currentResponseScope = payload.alfResponseScope;
```

Comment:
`this.currentResponseScope = payload.responseScope || payload.alfOriginScope;` ???